src/chrome/content/rules/jekyllrb.com.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+    Redirection:
+        www.jekyllrb.com -> jekyllrb.com
+-->
+<ruleset name="Jekyll">
+    <target host="jekyllrb.com" />
+    <target host="www.jekyllrb.com" />
+
+    <securecookie host="^(www\.)?jekyllrb\.com$" name=".+" />
+
+    <!-- Fix this redirection -->
+    <rule from="^http(s)?://jekyllrb.com/"
+            to="https://jekyllrb.com/" />
```

Comment:
I don't think line 12-13 are necessary. Perhaps you mean the `https://www.jekyllrb.com -> http://jekyllrb.com` issue? In the end, this ruleset would still redirects user to https://jekyllrb.com, even without line 12-13, due to line 6.